tests/test_logout_functionality.py

```diff
@@ -24,6 +28,6 @@ def test_logout_functionality(self, login_to_saucedemo):
         driver.get(restricted_page_url)
         WebDriverWait(driver, 10).until(EC.url_to_be("https://www.saucedemo.com/"))
 
-        print("Test passed:User successfully log out of the application")
+        logging.info("Test passed:User successfully log out of the application")
```

Comment:
@Sv-24 could you please add some verification that user logout? for example check that it is login page.